assets/containerisableComponents/marketingConsent/marketingConsent.jsx

```diff
@@ -0,0 +1,79 @@
+// @flow
+
+// ----- Imports ----- //
+
+import * as React from 'react';
+import CtaLink from 'components/ctaLink/ctaLink';
+import CheckboxInput from 'components/checkboxInput/checkboxInput';
+import ErrorMessage from 'components/errorMessage/errorMessage';
+import DotcomCta from 'components/dotcomCta/dotcomCta';
+import PageSection from 'components/pageSection/pageSection';
+
+import type { Csrf as CsrfState } from 'helpers/csrf/csrfReducer';
+
+
+// ----- Types ----- //
+
+type PropTypes = {
+  consentApiError: boolean,
+  confirmOptIn: ?boolean,
+  onClick: (marketingPreferencesOptIn: boolean, email?: ?string, csfr: CsrfState) => void,
+  marketingPreferencesOptIn: boolean,
+  marketingPreferenceUpdate: (preference: boolean) => void,
+  email?: ?string,
+  csrf: CsrfState,
+};
+
+// ----- Component ----- //
+
+const MarketingConsent = (props: PropTypes): React.Node => {
+  let content = null;
+  if (!props.email) {
+    return content;
+  }
+
+  if (props.confirmOptIn === null && props.email !== null && props.email !== undefined) {
+    content = (
+      <PageSection
+        modifierClass="marketing-consent"
+        heading="Stay in touch"
+      >
+        <CheckboxInput
+          id="gnm-marketing-preference"
+          checked={props.marketingPreferencesOptIn || false}
```

Comment:
This isn't an optional prop, do you need the `|| false` here?

Reply:
Good catch!!!